Microsoft.Toolkit.Uwp.UI.Controls/RemoteDevicePicker/RemoteDevicePicker.cs

```diff
@@ -0,0 +1,191 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Toolkit.Uwp.Helpers;
+using Windows.System.RemoteSystems;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    /// <summary>
+    /// Picker Control to show List of Remote Devices that are accessible
+    /// </summary>
+    [TemplatePart(Name = "PART_ListDevices", Type = typeof(ListView))]
+    [TemplatePart(Name = "PART_ListDeviceTypes", Type = typeof(ComboBox))]
+    [TemplatePart(Name = "PART_ProgressRing", Type = typeof(ProgressRing))]
+    public sealed class RemoteDevicePicker : ContentDialog
+    {
+        private ListView _listDevices;
+        private ComboBox _listDeviceTypes;
+        private ProgressRing _progressRing;
+        private Grid _commandSpace;
+
+        /// <summary>
+        /// Gets or sets List of All Remote Systems based on Selection Filter
+        /// </summary>
+        private ObservableCollection<RemoteSystem> RemoteSystems { get; set; }
+
+        /// <summary>
+        /// Gets or sets the Line Color on control Header. takes **SystemControlBackgroundAccentBrush** by default
+        /// </summary>
+        public Brush HeaderLineBrush
+        {
+            get { return (Brush)GetValue(HeaderLineBrushProperty); }
+            set { SetValue(HeaderLineBrushProperty, value); }
+        }
+
+        /// <summary>
+        /// Gets the dependency property for <see cref="HeaderLineBrush"/>.
+        /// </summary>
+        public static readonly DependencyProperty HeaderLineBrushProperty = DependencyProperty.Register(
+            nameof(HeaderLineBrush),
+            typeof(Brush),
+            typeof(RemoteDevicePicker),
+            new PropertyMetadata(Application.Current.Resources["SystemControlBackgroundAccentBrush"]));
```

Comment:
The default should be moved to the template, don't use Application.Current.Resources